tests/test_invalid_languages.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-from src.main import translate_pgn_move_token
```

Comment:
Couple of suggested changes to the folder structure:
* Move `translator` (the app folder) to an apps folder.  So your import for the routes will look something like this...
`from app.pgn_translator import views`

* Move the module to a folder so that your imports look something like this...
`from app.pgn_translator.utils.translate_pgn import translate_pgn_move_token`